docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.8"
```

Comment:
I feel like this isn't the first time we've seen these quotation marks change. Do we have conflicting Prettier setups somewhere?

Reply:
Someone must be changing them to double, but we should be using single quotes.